src/modules/pushPair/models/PushPairConfig.ts

```diff
@@ -60,6 +69,8 @@ export function parsePushPairConfig(config: PushPairConfig): PushPairInternalCon
     return {
         ...config,
         id: `${config.type}-${config.networkId}-${pairIds.join(',')}`,
+        deviationPercentage: config.deviationPercentage ?? 0.5,
+        minimumUpdateInterval: config.minimumUpdateInterval ?? 1,
```

Comment:
Default value is 1ms?

```suggestion
        minimumUpdateInterval: config.minimumUpdateInterval ?? 60000,
```

Reply:
Yeah I've set the interval at 1 so I can make sure that it will always update after an interval trigger